ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,1136 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sp: <http://spinrdf.org/sp#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+<https://semopenalex.org/vocab#> a owl:Ontology;
+  rdfs:label "SemOpenAlex Ontology"@en;
+  rdfs:comment "The Semantic OpenAlex Ontology, described using W3C RDF Schema and the Web Ontology Language OWL."@en .
+
+<https://semopenalex.org/class/Author> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Author"@en .
+
+<https://semopenalex.org/class/Institution> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Institution"@en .
+
+<https://semopenalex.org/class/Work> a owl:Class;
+  owl:equivalentClass <http://purl.org/spar/fabio/Work>;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Work"@en .
+
+<https://semopenalex.org/class/ConceptScore> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "ConceptScore"@en .
+
+<https://semopenalex.org/class/CountsByYear> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "CountsByYear"@en .
+
+<https://semopenalex.org/class/OpenAccess> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "OpenAccess"@en .
+
+<https://semopenalex.org/class/Concept> a owl:Class;
+  rdfs:subClassOf skos:Concept;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Concept"@en .
+
+<https://semopenalex.org/class/Geo> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Geo"@en .
+
+<https://semopenalex.org/class/Publisher> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Publisher" .
+
+<https://semopenalex.org/class/AuthorPosition> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "AuthorPosition"@en .
```

Comment:
do you maybe want to use spaces in the label, i.e. "Author Position"?

Reply:
it was developed by AIFB. i will leave as it for now.